File: Logic & Design/Visual Studio Code/Funcitons/Java/areaRectangleFunction.java
/*
 * Programmer:      Corey Jenkins
 * Date:            August 15, 2023
 * Filename:        areaRectangleFunction.java
 * Purpose:         A program in which returns the area of a rectangle.
*/

import java.util.Scanner;

public class areaRectangleFunction {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int rectangleLength, rectangleWidth, areaReturnFunction;
        System.out.println("Enter the length of a rectangle: ");
        rectangleLength = input.nextInt();

        System.out.println("Enter the height of the rectangle: ");
        rectangleWidth = input.nextInt();

        areaReturnFunction = area(rectangleLength, rectangleWidth);
        System.out.println(areaReturnFunction);

    }

    public static int area(int num1, int num2) {
        int areaRectangle;
        areaRectangle = num1 * num2;
        return areaRectangle;
    }
}

File: Logic & Design/Visual Studio Code/Funcitons/Python/area_rectangle_function.py
'''
Programmer:     Corey Jenkins
Date:           August 15, 2023
Filename:       area_rectangle_function.py
Purpose:        A program in which returns the area of a rectangle.
'''

def main():
    rectangle_length = int(input("Enter the length:"))
    rectangle_width = int(input("Enter the widtht:"))
    area_function = area(rectangle_length, rectangle_width)
    print("Area is", area_function)
    
# Function. Will multiply to give the area and will return the results back to area_function in the Main module.
def area(num1, num2):
    result = (num1 * num2)
    return result
    
main()